modin/pandas/test/test_io.py

```diff
@@ -69,6 +70,20 @@
 if not os.path.exists(IO_OPS_DATA_DIR):
     os.mkdir(IO_OPS_DATA_DIR)
 
+str_single_element = "1"
+str_single_col = """1
+2
+3
+"""
+str_four_cols = """1, 2, 3, 4
+5, 6, 7, 8
+9, 10, 11, 12
+"""
+str_non_unique_cols = """col,col,col,col
+5,6,7,8
+9,10,11,12
+"""
+
```

Comment:
It's quite specific, move it into test function.

Reply:
Tests parametrized using these variables, so now we can't move strings into function.

Reply:
We should not define test data that is needed in one test in the global scope.

Try to rewrite.

Reply:
Corrected.